ocs_ci/krkn_chaos/noobaa_chaos_helper.py

```
@@ -457,6 +459,24 @@ def validate_noobaa_endpoints_health():
 # ============================================================================
 
 
+def get_noobaa_db_chaos_target():
+    """
+    Return NooBaa DB label selector and primary container for chaos tests.
+
+    Matches deployment layout by OCS version (pre-4.7 mongo, 4.7–4.18 StatefulSet
```

Ruff (0.15.14)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)